styleguidekit-assets-altinn/dist/styleguide/css/styleguide.css

```diff
@@ -1383,7 +1383,7 @@
   font-size: 100%;
 }
 .sg-annotations-list-title:after {
-  content: "—";
+  content: "ÔÇö";
```

Comment:
Denne må også fikses.

Reply:
@clarkt . Endret encoding for den type av content til ISO